src/firmware/boards/legacy_robot_stm32f4/tick.c

```diff
@@ -129,7 +129,10 @@ static void normal_task(void *UNUSED(param))
                 current_primitive_index = 99;
             }
 
-            record->tick.drive_serial    = receive_last_serial();
+            // We no longer use message serials, but this is kept so that the message
```

Comment:
Can we get an issue # to potentially remove this at some point?

Reply:
I honestly don't even think it's worth the effort doing this at any point before we deprecate the old robots.